packages/builder/src/pages/builder/portal/apps/index.svelte

```diff
@@ -260,8 +266,14 @@
     }
   }
 
+  async function fetchTemplates() {
+    const response = await api.get("/api/templates?type=app")
```

Comment:
NAB: It might be a nice idea to save templates in a store. On initial load we need to load them so we can't avoid a loading spinner there, but if we cache them in a store (and still refresh them on mount here) it means that at least after the initial load there won't be a flashing loading spinner! I haven't checked out the code at ran it though so maybe that's a non-issue - it might look totally fine already.

Reply:
Seemed fine, but added a templates store anyways, slow internet you'd maybe run into this.